week-2/implement/bmi.js

```diff
@@ -13,3 +13,10 @@
 // Given someone's weight in kg and height in metres
 // Then when we call this function with the weight and height
 // It should return their Body Mass Index to 1 decimal place
+
+function massIndex(weight,height){
+    return (Number(height * height) / weight).toFixed(1);
```

Comment:
This seems to work but the call to Number doesn't do anything here. Number converts non numbers into numbers, but it does nothing here. The result of height * height will always be either a number (if height was a number) or NaN (if height was not a number).Other than this, it looks good!